.env.example

```
@@ -15,6 +15,7 @@ VIDEO_TRANSCODE_BSKY_DOMAIN_LIST = "video.fxbsky.app"
 POLYGLOT_DOMAIN_LIST = "polyglot.fxembed.com"
 POLYGLOT_ACCESS_TOKEN = ""
 BLUESKY_API_HOST_LIST = "api.fxbsky.app"
+GENERIC_API_HOST_LIST = "api.fxembed.com"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)